src/services/appFirebase.js

```diff
@@ -15,4 +15,9 @@ const githubAuthProvider = new firebase.auth.GithubAuthProvider();
 githubAuthProvider.addScope('gist');
 githubAuthProvider.addScope('public_repo');
 
-export {auth, database, githubAuthProvider};
+const googleAuthProvider = new firebase.auth.GoogleAuthProvider();
+googleAuthProvider.addScope('https://www.googleapis.com/auth/classroom.courses.readonly');
+googleAuthProvider.addScope('https://www.googleapis.com/auth/classroom.coursework.students');
+googleAuthProvider.addScope('https://www.googleapis.com/auth/classroom.coursework.me');
```

Comment:
This information is duplicated in the `gapi` service below—better to have one module export a constant and the other import it, or extract a shared configuration module?